app/javascript/components/EditEvent.js

```diff
@@ -63,31 +50,6 @@ class EditEvent extends React.Component {
       this.setState({ eventAttrs: response });
     });
   }
-  // componentDidMount = () => {
-  //   /* global fetch */
-  //   fetch("/events/:id")
-  //   console.log(this.props.match.params.id)
-  // }
-  
-  // componentDidUpdate = (currentEvent) => {
-  //   if(currentEvent.match.params.id != this.props.match.params.id){
-  //     this.getEvent();
-  //   }
-  // }
-  
-  // onChange = (e) => {
-  //   const{eventUpdate} = this.state
-  //   console.log(e.target.value)
-  //   eventUpdate[e.target.name] = e.target.value
-  // }
-  
-  // localSubmit = (e) => {
-  //   const {editEvent} = this.props
-  //   const {eventId, eventUpdate} = this.state
-  //   e.preventDefault(),
-  //   editEvent(eventId, eventUpdate)
-  //   window.location.href = '/Event'
-  // }
```

Comment:
Why all the commented code?  Let's just delete it.  Git has you covered if you want it back